include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -3697,6 +3697,9 @@ namespace internal
       const unsigned int                            face_no,
       const MatrixFreeFunctions::ShapeInfo<Number> &shape_info)
     {
+      if (dim == 1)
```

Comment:
Question: is it possible to have a 1D RT? If not one should probably also add here an assert!

Reply:
I think it is asserted in the class itself, but I might be mistaken

Reply:
There is no assert here:

https://github.com/dealii/dealii/blob/ecdd5f268b0da3a1ae4c23d974fab1b870f249f1/include/deal.II/matrix_free/evaluation_kernels.h#L3555-L3557

Reply:
This is the one I was referring to 
https://github.com/dealii/dealii/blob/ecdd5f268b0da3a1ae4c23d974fab1b870f249f1/source/fe/fe_raviart_thomas_nodal.cc#L445